lib/task/import/csvAccessionImportTask.class.php

```diff
@@ -416,18 +416,17 @@ public function execute($arguments = [], $options = [])
                     $self->status['resourceTypes'] = self::refreshTaxonomyTerms(QubitTaxonomy::ACCESSION_RESOURCE_TYPE_ID);
                 }
 
-                $this->setObjectPropertyToTermIdLookedUpFromTermNameArray(
-                    $self,
-                    'resourceTypeId',
+                $self->object->resourceTypeId = $self->translateNameToTermId(
                     'resource type',
                     $data,
```

Comment:
Sorry, I'm a bit confused, the signature for `QubitFlatfileImport::translateNameToTermId()` expects a string for the second argument.  Is `$data` a string value? If so, it's a very confusing variable name. :(

Reply:
Yes it seems to be and yes I agree - I had to take a good look at what was going on there.